hazelcast/src/test/java/com/hazelcast/client/cache/impl/nearcache/ClientCacheNearCacheCacheOnUpdateTest.java

```diff
@@ -103,14 +104,14 @@ private static void runTest(ICache<Integer, Integer> icacheOnClient,
         Runnable getter = () -> {
             int i = 0;
             while (!stop.get()) {
-                icacheOnClient.get(i++ % NUM_OF_KEYS);
+                icacheOnClient.get(++i % NUM_OF_KEYS);
```

Comment:
notice that now we get items starting from 1 instead of 0, is it ok to change this? If it does not matter, ignore this comment.

Reply:
it does not really matter since it rotates among the 100 items.